setup.py

```diff
@@ -22,7 +22,7 @@
 name = 'google-cloud-securitycenter'
 description = 'Cloud Security Command Center API API client library'
 version = "0.4.0"
```

Comment:
Bump to 1.0.0?

Reply:
Release-please will bump it. :) A new path helper has been added since the last noted release (#25). It is not an API feature though so I think it is alright to release the 1.0